modules/@apostrophecms/doc-type/index.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
         slug: {
           type: 'slug',
           label: 'Slug',
-          following: 'title',
+          following: [ 'title', 'archived' ],
```

Comment:
I made this change on all that were setting `following` for the slug. I have an update ready for the docs as well, adding a tip to do this if you're changing the slug's following property.

We might instead consider adding `archived` automatically always (we take it out of the slug input component where it mashes strings together). That would reduce the opportunity to screw this up.